src/plugins/intel_npu/src/al/include/npu_private_properties.hpp

```diff
@@ -0,0 +1,347 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/runtime/intel_npu/properties.hpp"
+
+namespace ov {
+namespace intel_npu {
```

Comment:
I see that different namespaces are used across these files:
![image](https://github.com/openvinotoolkit/openvino/assets/2566854/24a790ef-9701-4c83-8734-d9a52b221431)
![image](https://github.com/openvinotoolkit/openvino/assets/2566854/0af278ff-ebe0-4cf2-b9fd-0b142f3eb6d2)
can we unify it?